### PR TITLE
[Term Entry] Kotlin Strings .compareTo()

### PR DESCRIPTION
**Pull request by Fejiro001**
### Description

- Added the .compareTo() method for Kotlin Strings
- Closes Issue #3721 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/terms/compareTo/compareTo.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+Title: 'compareTo()'
```
Comment (ishg-153) [inline content/kotlin/concepts/strings/terms/compareTo/compareTo.md:2]:
```suggestion
Title: '.compareTo()'
```

File: content/kotlin/concepts/strings/terms/compareTo/compareTo.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`compareTo()`** method in Kotlin is used to compare two strings lexicographically, meaning it compares the strings character by character based on their Unicode values. It helps determine the relative order of two strings in a dictionary or alphabetically. This method is commonly employed for sorting strings or performing string comparisons.
```
Comment (ishg-153) [inline content/kotlin/concepts/strings/terms/compareTo/compareTo.md:15]:
```suggestion
The **`.compareTo()`** method in Kotlin is used to compare two strings lexicographically, meaning it compares the strings character by character based on their Unicode values. It helps determine the relative order of two strings in a dictionary or alphabetically. This method is commonly employed for sorting strings or performing string comparisons.
```

File: content/kotlin/concepts/strings/terms/compareTo/compareTo.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+## Syntax
+
+```pseudo
+stringOne.compareTo(stringTwo)
+```
+
+- `stringOne`: The first string to be compared.
+- `stringTwo`: The second string to be compared.
+
+If `stringOne` is comes before `stringTwo` lexicographically, `compareTo` returns a negative value. If they are equal, it returns `0`. If `stringOne` comes after `stringTwo` lexicographically, `compareTo` returns a positive value.
```
Comment (ishg-153) [inline content/kotlin/concepts/strings/terms/compareTo/compareTo.md:26]:
```suggestion
If `stringOne` comes before `stringTwo` lexicographically, `.compareTo()` returns a negative value. If they are equal, it returns `0`. If `stringOne` comes after `stringTwo` lexicographically, `.compareTo()` returns a positive value.
```

File: content/kotlin/concepts/strings/terms/compareTo/compareTo.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Example
+
+This example shows how to use the `compareTo()` method to two strings:
```
Comment (ishg-153) [inline content/kotlin/concepts/strings/terms/compareTo/compareTo.md:30]:
```suggestion
This example shows how to use the `.compareTo()` method to two strings:
```

File: content/kotlin/concepts/strings/terms/compareTo/compareTo.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+        println("$stringOne comes before $stringTwo")
+    } else if (result == 0) {
+        println("$stringOne and $stringTwo are the same")
+    } else {
+        println("$stringOne comes after $stringTwo")
+    }
+}
+```
+
+The result of the example is:
```
Comment (ishg-153) [inline content/kotlin/concepts/strings/terms/compareTo/compareTo.md:49]:
```suggestion
The result of this example is:
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Fejiro001 I have suggested some changes, please make those.

**Comment by Fejiro001** [comment]
ok @ishg-153

**Review by ishg-153 — APPROVED** [review]
This looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@Fejiro001 LGTM

**Pull request merged**